doc/crypto/api/ops/ka.rst

```diff
@@ -9,11 +9,19 @@
 Key agreement
 =============
 
```

Comment:
Missing change: in `doc/crypto/api/keys/policy.rst`: “This flag must be present on keys used with the following APIs: …”

Reply:
Thank you. I had forgotten about these cross references.